src/Specular/FRP/Base.purs

```diff
@@ -268,6 +270,9 @@ filterMapEvent f = filterMapEventB (pure <<< f)
 filterEvent :: forall a. (a -> Boolean) -> Event a -> Event a
 filterEvent f = filterMapEvent (\x -> if f x then Just x else Nothing)
 
+filterJustEvent :: forall a. Event (Maybe a) -> Event a
```

Comment:
This is rather analogous to `catMaybes`. Name it `catMaybesEvent`?

Reply:
I see there's precedent: https://www.stackage.org/haddock/lts-14.12/threepenny-gui-0.8.3.0/Graphics-UI-Threepenny-Core.html#v:filterJust

Reply:
I feel more like "filtering" events than "concatenating" them.